docs/docs-content/integrations/maintenance-policy.md

```diff
@@ -10,93 +10,115 @@ tags: ["packs", "deprecation"]
 
 Palette supports two pack categories: *Infrastructure* and *Add-on*. Infrastructure packs are often sourced from third parties and are infrastructure-related or support critical container-based environments. Packs are used to create layers in cluster profiles. Infrastructure packs are grouped as follows:
 
-- Kubernetes
-
 - Operating System (OS)
-
+- Kubernetes
 - Container Network Interface (CNI)
-
 - Container Storage Interface (CSI)
 
 
 Add-on packs provide additional functionality that you can add to your cluster profile and are grouped as follows:
```

Comment:
Do we need to capitalize the initial letter in this list? I see that our pack list in documentation do not have these chaptalized. These are not proper nouns, so we probably don't need to capitalize.

Reply:
Since these are acronyms and are initial-capped in much of the documentation, I'll leave them as is for now.